samples/subsys/fs/littlefs_blk/boards/nucleo_h743zi.overlay

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021 Lukasz Majewski, DENX Software Engineering GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&sdmmc1 {
+	status = "okay";
+	pinctrl-0 = <&sdmmc1_d0_pc8
+		     &sdmmc1_ck_pc12
+		     &sdmmc1_cmd_pd2>;
+
+	pinctrl-names = "default";
+};
```

Comment:
As I said, there is no reason to have someone specific hardware support in Zephyrs tree.

Reply:
@jfischer-no  What do you mean by `someone specific hardware support` ?

To test this - you need `Nucleo H743ZI` board and connect SD card's D0, CLK and CMD pins to it with the PC8, PC12 and PD2. That is the HW which I do have.

Unfortunately, I don't have any board with both mounted SPI-NOR and SD card.